qt/qt_common/map_widget.cpp

```diff
@@ -267,6 +267,10 @@ void MapWidget::Build()
                            QVector4D(-1.0, -1.0, 0.0, 0.0),
                            QVector4D(1.0, -1.0, 1.0, 0.0)};
   m_vbo->allocate(static_cast<void*>(vertices), sizeof(vertices));
+  QOpenGLFunctions *f = QOpenGLContext::currentContext()->functions();
+  f->glEnableVertexAttribArray(0);
+  f->glVertexAttribPointer(0, 4, GL_FLOAT, GL_FALSE, 4 * sizeof(GLfloat), nullptr);
+  m_vbo->release();
```

Comment:
I'm not an expert, but is m_vbo->release() needed here and why it was absent before? Is it somehow connected with a bug?

- add a comment that 0-index is "a_position"
- add a reference to this PR in the code.
- add a reference to the issue in PR's description

Reply:
Added comment about "a_position" binding. But as a novice in OpenGL it's possible that wording should be improved.

Added cross references.

Reply:
By the way `m_vbo->release()` call could be immited. Don't know if we need it. OpenGL experts required.

Reply:
Removed.